project_wbs/view/project_project_view.xml

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--KANBAN VIEW-->
+    <record id="view_project_kanban" model="ir.ui.view">
+        <field name="name">project.project.kanban.wbs</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_kanban"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="analytic_account_id" invisible="1"/>
+            </field>
+            <xpath expr="//templates" position="before">
+                <field name="project_child_complete_ids"/>
+            </xpath>
+            <xpath expr="//div[hasclass('o_primary')]" position="inside">
+                <a type="object" name="action_open_view_project_form">
+                <div class="oe_wbs_subtitle">
+                    <div class="parentheading">
+                        <field name="account_class"/>
+                        <field name="complete_wbs_code"/>
+                    </div>
+                </div>
+                </a>
+                <div class="oe_nav_buttons"
+                     name="wbs_navigation_up"
+                     attrs="{'invisible': [('parent_id', '=', False)]}">
+                    <a name="action_open_parent_kanban_view"
+                       type="object">
+                        <!--GO UP-->
+                        <div class="oe_wbs_navigation">
+                            <div class="oe_tag">
+                                (Parent: <field name="parent_id"/>)
+                            </div>
+                            <div class="fa fa-level-up text-center parentheading oe_kanban_action_a"/>
+                        </div>
+                    </a>
+                </div>
+                <hr/>
+                <div class="oe_nav_buttons" name="wbs_navigation_down">
+                    <a name="action_open_child_kanban_view"
+                       type="object">
+                        <!--GO DOWN-->
+                            <div class="oe_wbs_navigation parentheading oe_kanban_action_a">
+                                    WBS Elements:
+                                    <field name="project_child_complete_ids"/>
+                            </div>
+                            <t t-foreach="record.project_child_complete_ids.raw_value" t-as='child'>
+                                <t t-esc="child.name"/>
```

Comment:
> Do not show full names in children in Kanban View

This could've been accomplished by using `complete_name` vs `name`, instead of overriding name_get()

Reply:
Ok, but the name_get is used in the wbs list also, that it's important.

Reply:
Sorry, didn't get it. In the list of projects you mean?

Reply:
Hi @alexey-pelykh, the purpose of overriding name_get has nothing to do with the kanban view. It is more about searching analytic accounts in any analytic_account_id field in any other place in Odoo. It is important for WBS to see quickly the full path code. Honestly,  I did not even think about name_get affecting this view. I really don't want that. Because, if we have  large structure with many levels, the complete name would be too long. Same as we put complete_wbs_name field. I will check that.